packaging/windows/OpenRA.nsi

```diff
@@ -129,13 +129,13 @@ Section "Game" GAME
 	File "${SRCDIR}\RedAlert.exe"
 	File "${SRCDIR}\TiberianDawn.exe"
 	File "${SRCDIR}\Dune2000.exe"
-	File "${SRCDIR}\OpenRA.Game.exe"
-	File "${SRCDIR}\OpenRA.Game.exe.config"
-	File "${SRCDIR}\OpenRA.Utility.exe"
-	File "${SRCDIR}\OpenRA.Server.exe"
+	File "${SRCDIR}\OpenRA.exe"
+	File "${SRCDIR}\OpenRA.Game.dll"
+	File "${SRCDIR}\Utility.exe"
+	File "${SRCDIR}\Server.exe"
 	File "${SRCDIR}\OpenRA.Platforms.Default.dll"
 	File "${SRCDIR}\ICSharpCode.SharpZipLib.dll"
-	File "${SRCDIR}\FuzzyLogicLibrary.dll"
+	;File "${SRCDIR}\FuzzyLogicLibrary.dll"
```

Comment:
We're back to commenting these out - why?

Reply:
because this have been moved to mods\common folder and this is covered by `File /r "${SRCDIR}\mods\common"` im guessing.